tests/daemon/test-daemon.c

```diff
@@ -1402,48 +1401,15 @@ test_daemon_refresh_and_reload_machine_id (Fixture       *fixture,
                                    NULL,
                                    initial_machine_id);
 
-  /* Overwrite machine-id */
-  emer_daemon_reset_tracking_id (fixture->test_object, &error);
-  g_assert_no_error (error);
-
-  /* New machine id should now be different */
-  emer_machine_id_provider_get_id (fixture->mock_machine_id_provider,
-                                   NULL,
-                                   new_machine_id);
-
-  g_assert (uuid_compare (initial_machine_id, new_machine_id) != 0);
-
-  /* Send some metrics using the newly created daemon. We'll assert
-   * one we receive the metrics that the received metric has the same
-   * machine-id as the one provided by the machine-id provider */
+  /* Send some metrics using the created daemon. We'll assert one we receive the
+   * metrics that the received metric has the same machine-id as the one
+   * provided by the machine-id provider */
```

Comment:
This message is no longer true. We don't make any assertion about the machine ID. Is this test redundant with another test which calls `record_singulars`, `assert_singulars_received`, `wait_for_upload_to_finish`?

Reply:
You are right!  This becomes redundant.